EventVisualisation/Base/src/ConfigurationManager.cxx

```diff
@@ -0,0 +1,61 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file    ConfigurationManager.cxx
+/// \author  Jeremi Niedziela
+///
+
+#include "EventVisualisationBase/ConfigurationManager.h"
+
+#include <TSystem.h>
+
+#include <iostream>
+
+using namespace std;
+
+namespace o2  {
+namespace EventVisualisation {
+  
+ConfigurationManager* ConfigurationManager::sInstance  = nullptr;
+
+ConfigurationManager* ConfigurationManager::getInstance()
+{
+  if(!sInstance){new ConfigurationManager();}
+  return sInstance;
+}
+
```

Comment:
Isn't the more proper way for a singleton something like:
`
static ConfigurationManager& getInstance()
{
  static ConfigurationManager manager;
  return manager;
`
If I remember correctly this should also be thread save.
I think you use singletons also in other places.
See discussion here
https://stackoverflow.com/questions/1008019/c-singleton-design-pattern
https://stackoverflow.com/questions/270947/can-any-one-provide-me-a-sample-of-singleton-in-c/271104#271104

Reply:
Thanks, I read this and updated singletons accordingly.